src/components/implementation/cbuf_mgr/default/cbuf_mgr.c

```diff
@@ -62,8 +73,8 @@ struct cbuf_maps {
 };
 
 struct cbuf_info {
-	u32_t cbid;
-	int size;
+	int cbid;
```

Comment:
Let's be careful here because cbuf_create does return a negative number sometimes. But that might not necessarily mean cbids should be signed. Just those return types, and the variables that handle them.